.gitlab-ci.yml

```diff
@@ -3262,6 +3227,8 @@ dev_branch_docker_hub-a7-windows:
     - docker_build_agent7_windows1809_jmx
     - docker_build_agent7_windows1909
     - docker_build_agent7_windows1909_jmx
+    - docker_build_agent7_windows2004
+    - docker_build_agent7_windows2004_jmx
   variables:
     VARIANT: 1909
```

Comment:
All publish jobs should use latest Windows worker to make sure Docker won't reject pulling the newer image right away (several occurrences in the PR)